### DISCUSSION TITLE
Best practice for deploying multiple brokers with security settings?

### DISCUSSION THREAD

**Discussion by andr0272** [arkmq-org/arkmq-org-broker-operator#474] [Q&A] [answered]
I am trying to deploy an operator with 3 brokers, and a security configuration using the ActiveMQAtremisSecurity CR. When I run this deployment I get a persistent warning on all 3 brokers saying "There are more than one servers on the network broadcasting the same node id". This seems to only happen when I deploy the brokers simultaneously. If I deploy a single broker with this configuration, than scale up by reapplying the ActiveMQArtemis CR with spec.deploymentPlan.size: 3, I get 3 seperate NodeIDs. Has anyone encountered this?
Edit: Version 1.0.5

**Comment by brusdev** [comment, marked as answer]
@andr0272 can you reproduce this issue on 1.0.7 and share your ActiveMQArtemis CR with the relative logs?

**Reply by andr0272** [reply]
Updating to 1.0.8 resolved my issue. 
Thank you for your guidance, sorry for leaving this open so long.
Best!

**Reply by brusdev** [reply]
@andr0272 thanks for your feedback

**Comment by Emmett-42** [comment]
Hi, I get this issue as well. I'm using the 2.1.4 version on my openshift cluster as the operator deployment.
```WARN` [org.apache.activemq.artemis.core.client] AMQ212034: There are more than one servers on the network broadcasting the same node id.```
 twice every few minutes on the ss-0 and the ss-1 pod keeps gracefully shutting down and restarting. the PVC are different. and when I scale down to 1 and then back to 2, the issue is still there, it doesn't recreate the nodeIDs.
any suggestions? what could be the cause of this. I also get reconcilation errors in the controller manager, could it be related to this - for the pod restarts?

**Reply by brusdev** [reply]
@Emmett-42 Apache Artemis reads the nodeID from the journal. If you see this error, it means that different PVCs include a journal with the same nodeID. You need to clean up or delete the affected PVCs.
